include/dicom/dicom-file.h

```diff
@@ -20,16 +21,17 @@ using Range = std::pair<size_t,size_t>;
 class DicomFile{
     friend class DicomAnonymizer;
     friend class TestAnonymizer;
-    using tag = uint64_t;
 private:
     std::shared_ptr<char[]> buffer;
     const void* data = nullptr;
     size_t size = 0;
 
-    std::vector<std::tuple<tag, Range>> elements;
+    std::vector<std::tuple<tag_uint64_t, Range>> elements;
+    std::unordered_map<tag_uint64_t, std::string> redacted_elements;
     bool is_valid = true;
 protected:
     void MakeHardlinks(const fs::path &master_path);
+    std::string GetData(tag_uint64_t tag);
```

Comment:
This might be a problem, in the feat-checksum branch, we have a function with the same name that returns the data pointer. Although it should still work with function overloading we should probably either rename the function here or in feat-checksum to avoid confusion in the future.

Reply:
I can rename this to GetElementValue. But should probably refactor the other one too. I'm sure I can find a way around needing that method. Like adding a calculate md5 to dicomfile

Reply:
it is done